flytekit/bin/entrypoint.py

```diff
@@ -158,14 +159,21 @@ def _dispatch_execute(
         logger.error(exc_str)
         logger.error("!! End Error Captured by Flyte !!")
 
-    # All the Non-user errors are captured here, and are considered system errors
+    # All the Non-user runtime errors are captured here, and are considered system errors
     except Exception as e:
-        exc_str = get_traceback_str(e)
+        kind = _error_models.ContainerError.Kind.RECOVERABLE
+        err = e
+
+        if isinstance(e, FlyteNonRecoverableSystemException):
```

Comment:
Can this be another `except FlyteNonRecoverableSystemException as e` branch?

Reply:
Good call. updated it